changelogs/unreleased/gh-4317-console-add-line-carrying-backslash.md

```diff
@@ -0,0 +1,5 @@
+## feature/console
+
+* Now multiline commands with all lines except last one ending with "\" can
+  be used instead of `\set delimiter "\"`, is active only without a set
```

Comment:
```
\set delimiter ;
```

1. No quotes.
2. Backslash here is confusing, I suggest to use a semicolon. The delimiter symbol is to mark end of a statement, not to mark its continuation. The opposite.

Reply:
Fixed.